docs/user/howto/manage-charms.md

```diff
@@ -212,13 +214,13 @@ Depending on the cloud substrate that your controller is running on, the above c
 
 ```{note}
 
-Depending on your use case, you may alternatively opt to provision a set of machines in advance via the `juju add-machine` command. 
+Depending on your use case, you may alternatively opt to provision a set of machines in advance via the `juju add-machine` command.
 
 In this case, when running the above `juju deploy` command, Juju will detect that the model contains machines with no applications assigned to them and automatically deploy the application to one of those machines instead of spinning up a new machine.
 
 ```
 
-The command also allows you to add another argument to specify a custom name (alias) for your deployed application (charms only). You can also take advantage of the rich set of flags to specify a charm channel or revision, a machine base, a machine constraint (e.g., availability zone), the number of application units you want (clusterised), a space binding, a placement directive (e.g., to deploy to a LXD container), a specific storage instance, a specific machine, etc., and even to trust the application with the current credential -- in case the application requires access to the backing cloud in order to fulfil its purpose (e.g., stojrage-related tasks). 
+The command also allows you to add another argument to specify a custom name (alias) for your deployed application (charms only). You can also take advantage of the rich set of flags to specify a charm channel or revision, a machine base, a machine constraint (e.g., availability zone), the number of application units you want (clusterised), a space binding, a placement directive (e.g., to deploy to a LXD container), a specific storage instance, a specific machine, etc., and even to trust the application with the current credential -- in case the application requires access to the backing cloud in order to fulfil its purpose (e.g., stojrage-related tasks).
```

Comment:
`specify a custom name (alias) for your deployed application (charms only)`

bundles allow you to specify a custom application name as well, inside of the yaml rather than on the cli.